### ISSUE TITLE
Doesn't seem to work with express-bunyan-logger

### ISSUE THREAD

**Issue by Limess** [benbria/bunyan-debug-stream#4]
Hi,

I'm trying to use a logger configured to use bunyan-debug-stream v1.0.5 with express-bunyan logger v1.1.1 but I get the following error:

```
req: Error running stringifier:
  ReferenceError: EXPRESS_BUNYAN_LOGGER_FIELDS is not defined
      at exports.stdStringifiers.req (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan-debug-stream/lib/BunyanDebugStream.js:287:36)
      at BunyanDebugStream._runStringifier (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan-debug-stream/lib/BunyanDebugStream.js:114:20)
      at BunyanDebugStream._entryToString (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan-debug-stream/lib/BunyanDebugStream.js:174:34)
      at BunyanDebugStream._write (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan-debug-stream/lib/BunyanDebugStream.js:237:28)
      at doWrite (_stream_writable.js:301:12)
      at writeOrBuffer (_stream_writable.js:288:5)
      at BunyanDebugStream.Writable.write (_stream_writable.js:217:11)
      at Logger._emit (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan/lib/bunyan.js:861:22)
      at Logger.info (/home/charlie/projects/o2/priority/app-web/node_modules/bunyan/lib/bunyan.js:960:24)
      at ServerResponse.logging (/home/charlie/projects/o2/priority/app-web/node_modules/express-bunyan-logger/index.js:143:23)
      at ServerResponse.emit (events.js:129:20)
      at ServerResponse.wrapped (/home/charlie/projects/o2/priority/app-web/node_modules/newrelic/lib/transaction/tracer/index.js:157:28)
      at ServerResponse.wrappedEmit [as emit] (/home/charlie/projects/o2/priority/app-web/node_modules/newrelic/lib/transaction/tracer/index.js:197:46)
      at finish (_http_outgoing.js:521:10)
      at afterWrite (_stream_writable.js:361:3)
      at _stream_writable.js:349:9
```

My config is as follows:

```
server.js

    var logger = require('./logger);
    var expressBunyanLogger = require('express-bunyan-logger');
    app.use(expressBunyanLogger({
      logger: logger
    }));

---

logger.js

var bunyanDebugStream = require('bunyan-debug-stream');
  logger = bunyan.createLogger({
    name: name,
    src: true,
    streams: [{
      level: 'info',
      type: 'raw',
      stream: bunyanDebugStream({
        basepath: path.join(__dirname, '../')
      })
    }],
    serializers: bunyanDebugStream.serializers
  });

```

Your README suggests compatibility, as does the projects source, so I was wondering if I'm missing something here.

**Comment by jwalton** [comment]
This appears to be broken.  I'll have a fix today.

**Comment by jwalton** [comment]
Fixed in v1.0.6.

**Comment by Limess** [comment]
Thanks
